eng/Versions.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
     <VersionPrefix>3.1.0</VersionPrefix>
-    <PreReleaseVersionLabel>preview3</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>preview4</PreReleaseVersionLabel>
```

Comment:
@mkArtakMSFT @wtgodbe Do you want to merge this change to aspnet/Blazor's blazor-wasm branch?

Reply:
Yes, this should be in blazor-wasm branch.